.gitignore

```diff
@@ -6,3 +6,4 @@
 java/Dockerfile
 php/Dockerfile
 python/Dockerfile
+serverless/Dockerfile
```

Comment:
The file is missing a `\n` at the end, it makes the diff look funny.

Reply:
Empty line has been added